libs/core/executors/tests/unit/standalone_thread_pool_executor.cpp

```diff
@@ -180,8 +181,8 @@ int main()
         hpx::threads::thread_pool_init_parameters thread_pool_init("my_pool", 0,
             hpx::threads::policies::scheduler_mode::default_mode, num_threads,
             0, notifier, ad, network_callback, 0,
-            (std::numeric_limits<std::int64_t>::max)(),
-            (std::numeric_limits<std::int64_t>::max)());
+            (std::numeric_limits<std::int64_t>::max) (),
+            (std::numeric_limits<std::int64_t>::max) ());
```

Comment:
Have you switched to clang-format v12 in this repo?

Reply:
Nope, just my local clang-format being different. I'll fix this.